Assets/SEE/Controls/Actions/ShowCodeAction.cs

```diff
@@ -185,6 +186,52 @@ static int GetAttribute(Edge edge, string attribute)
             }
         }
 
+        /// <summary>
+        /// Returns a new CodeWindow showing a diff for the given edge in a diff city.
+        /// </summary>
+        /// <param name="edgeRef">The edge to get the CodeWindow for</param>
+        /// <returns>new CodeWindow showing a diff</returns>
+        public static CodeWindow ShowDiff(GraphElementRef graphElementRef, AbstractSEECity city)
+        {
+            GraphElement graphElement = graphElementRef.Elem;
+            (string sourceFilename, string path) = GetPath(graphElement);
+            DiffCity diffCity = city.gameObject.MustGetComponent<DiffCity>();
```

Comment:
I am confused. A `city` containing another `DiffCity`? This cannot be true. The argument `city` should be the `DiffCity`.

Reply:
You are right, should be fixed now.

Reply:
Fine, except that the documentation of the method was incomplete and had copy&paste issues that missed adaptations. I fixed it.